src/cli/pipeline.rs

```diff
@@ -55,6 +62,56 @@ pub fn decode<P: AsRef<Path>>(f: P) -> Result<Image, ImageErrors> {
                 return Image::from_decoder(decoder);
             };
 
+            #[cfg(feature = "tiff")]
+            if f.as_ref()
+                .extension()
+                .is_some_and(|f| f.eq_ignore_ascii_case("tiff") | f.eq_ignore_ascii_case("tif"))
+            {
+                let mut decoder = tiff::decoder::Decoder::new(file).map_err(|e| {
```

Comment:
Move this to codecs module. We have exported modules for other users of zune-image so they can use ours codecs